server/routerlicious/packages/routerlicious-base/src/alfred/index.ts

```diff
@@ -3,11 +3,12 @@
  * Licensed under the MIT License.
  */
 
+export { IAlfredResourcesCustomizations } from "./customizations";
 export { AlfredRunner } from "./runner";
 export {
 	AlfredResources,
 	AlfredResourcesFactory,
 	AlfredRunnerFactory,
 	OrdererManager,
 } from "./runnerFactory";
-export { DeltaService } from "./services";
+export { DeltaService, StorageNameAllocator } from "./services";
```

Comment:
nit: Does StorageNameAllocator lives in ./service folder? I see it present in services-core/src folder.